gcs/src/gcs.cpp

```diff
@@ -2692,6 +2741,7 @@ _set_sync_donor (gcs_conn_t* conn, const char* value)
 
         conn->params.sync_donor = sd;
         conn->max_fc_state      = sd ? GCS_CONN_DONOR : GCS_CONN_JOINED;
+        gu_config_set_bool (conn->config, GCS_PARAMS_SYNC_DONOR, sd);
```

Comment:
why this change?

Reply:
In all other param setters here we update the config. I don't know how it work in details, because `conn->config` isn't used in GCS. Either this is purposeless and we should delete updating it everywhere in gcs.cpp setters, or this config is shared between modules and hence should be fresh. But I did this for uniformity with other setters: they all update it.

Reply:
Anyway, reverted this, because it's from upstream and don't seem significant.